error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/util/MoreMatchers.java

```diff
@@ -0,0 +1,34 @@
+package tech.picnic.errorprone.bugpatterns.util;
+
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.predicates.TypePredicate;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.Tree;
+import com.sun.tools.javac.code.Symbol;
+
+/** A collection of methods to enhance the use of {@link Matcher}s. */
```

Comment:
Here we are missing a link to `Matchers` (not to mix up with `Matcher`s that is used in the line above 😅).

Will add it.